pom.xml

```diff
@@ -40,15 +40,15 @@
     <changelist>-SNAPSHOT</changelist>
 
     <!-- jenkins versions -->
-    <jenkins.version>2.332.4</jenkins.version>
-    <bom.artifactId>bom-2.332.x</bom.artifactId>
-    <bom.version>1763.v092b_8980a_f5e</bom.version>
+    <jenkins.version>2.401.1</jenkins.version>
+    <bom.artifactId>bom-2.401.x</bom.artifactId>
+    <bom.version>2244.vd60654536b_96</bom.version>
```

Comment:
When initially bumping kubernetes-client-api, it needed a dependency on snakeyaml-api 2.0 and Jenkins needed to be at least 2.361.4 or higher.